File: src/main/java/com/manideepla/bookerang/BookerangApplication.java
package com.manideepla.bookerang;


import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class BookerangApplication {

	private static final Logger log = LoggerFactory.getLogger(BookerangApplication.class);

	public static void main(String[] args) {
		log.info("Starting BookerangApplication");
		SpringApplication.run(BookerangApplication.class, args);
	}

}

File: src/main/kotlin/com/manideepla/bookerang/BookerangApplication.kt
package com.manideepla.bookerang

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jdbc.repository.config.EnableJdbcRepositories

@EnableJdbcRepositories
@SpringBootApplication
class BookerangApplication

fun main(args: Array<String>) {
    runApplication<BookerangApplication>(*args)
}
